server.js

```diff
@@ -0,0 +1,78 @@
+const { client, syncAndSeed } = require ('./db')
+const express = require ('express')
+const path = require ('path')
+
+const app = express()
+
+app.use('/assets', express.static(path.join(__dirname, 'assets')))
+
+app.get('/', async (req, res, next) => {
+    try {
+        const response = await client.query('SELECT * FROM Book;')
+        const books = response.rows
+        res.send(`
+            <html>
+                <head>
+                    <link rel='stylesheet' href='/assets/styles.css' />
+                </head>
```

Comment:
see how you used this block of code in several places? you can pull it out to a separate variable like 
```js
const head = `
<head>
  <link rel='stylesheet' href='/assets/styles.css' />
</head>
`;

and when you use it: 
res.send(`
<html>
  ${head}
  <body>
    .. the rest of your code
  </body>
</html>
`);
```